unity/Assets/Scripts/Content/ContentData.cs

```diff
@@ -761,15 +759,32 @@ public class ContentPack
     // Get a unity texture from a file (dds or other unity supported format)
     public static Texture2D FileToTexture(string file)
     {
+        if (file == null) throw new System.ArgumentNullException("file");
         return FileToTexture(file, Vector2.zero, Vector2.zero);
     }
 
     // Get a unity texture from a file (dds or other unity supported format)
     // Crop to pos and size in pixels
     public static Texture2D FileToTexture(string file, Vector2 pos, Vector2 size)
     {
-        string imagePath = @"file://" + file;
-        WWW www = null;
+        if (file == null) throw new System.ArgumentNullException("file");
+        if (Application.platform == RuntimePlatform.Android)
+        {
+            if (!File.Exists(file))
+            {
+                string prefix = Path.Combine(Path.GetDirectoryName(file), Path.GetFileNameWithoutExtension(file));
```

Comment:
While this is OK, if we are doing something in the ini files to handle different file names, might not need this.  I'm not as concerned with this usage though.

Perhaps find/replace ".dds" with "" in the ini data and just try all supported extensions?

Reply:
Jepp, I think that is the best approach. Just use the image asset name without file extension in the config. And try out all supported extensions on image loading.